app/channels/application_cable/connection.rb

```diff
@@ -0,0 +1,8 @@
+module ApplicationCable
+  class Connection < ActionCable::Connection::Base
+    identified_by :current_user
+    def connect
+      self.current_user = User.find_by user_name: cookies.encrypted["_markus_session_csc108"]["real_user_name"]
```

Comment:
Instead of accessing the cookies directly, can you do `request.session[:user_name]`? This is what we use in `SessionHandler`, but it seems like we only have direct access to the `request` object in here?